libqtile/notify.py

```diff
@@ -23,109 +23,128 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-# type: ignore
```

Comment:
I'd removed the blanket mypy ignore but the result is that we now need lots of `# type: ignore` lines. Might be neater to go back to ignoring the whole file.

Reply:
I think this is preferable -- might as well type check as much as we can.